db/migrate/20200205151906_make_subscription_secret_encrypted.rb

```diff
@@ -0,0 +1,5 @@
+class MakeSubscriptionSecretEncrypted < ActiveRecord::Migration[5.2]
+  def change
+    rename_column :subscriptions, :secret, :encrypted_secret
```

Comment:
It would have been nicer to have this in the original migration. Also I'm not sure whether 'string' is the correct datatype for encrypted fields? I've often has to use binary-type DB fields in tha past

Reply:
@starswan agreed 1 migration would be simpler 👍 
Regarding the datatype, the output from ActiveSupport::MessageEncryptor is a (base64?) string, and so the string datatype is perfectly suited:
```
Encryptor.encrypt('foo')
=> "0AK1bcbDunemQ5akCQ==--huW00cOaMYxsWWuO--vnr6Px1ghnOkjUpdQbV+eA=="
```

Reply:
Ok that wasn't obvious to me. Maybe a comment is required?

Reply:
@starswan a comment will be added 👍